gammapy/background/models.py

```diff
@@ -24,6 +24,92 @@
 DEFAULT_SPLINE_KWARGS = dict(k=1, s=0)
 
 
+def add_column_and_sort_table(sources, pointing_position):
```

Comment:
To make a function or class appear in the API docs, you have to add it to the `__all__` list at the top of the file.

In this case though I think it's more an internal helper function that doesn't need to be advertised to end users as part of the public Gammapy API, i.e. my suggestion would be to not add it to `__all__` and maybe even to add an underscore to the function name to make it clear it's a private helper function: `_add_column_and_sort_table` (the underscore is the common patter to mark things private in Python.

@JouvinLea - What do you think? OK to keep this private?